build-scripts/src/index.ts

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env node
+import fs, { promises as fsp } from 'fs';
+import { program } from 'commander';
+import queryGitHub from './github';
+import querySecurityScorecard from './security-scorecard';
+
+program
+  .version('1.0.0', '-v, --version')
+  .requiredOption('-p, --path <string>', 'The path where the files should be saved.')
+  .requiredOption('-o, --org <string>', 'The github organisation.')
+  .requiredOption('--gh_priv_key <string>', 'The github private key for the app.')
+  .requiredOption('--gh_app_id <string>', 'The github id of the app.')
+  .requiredOption('--gh_app_install_id <string>', 'The github id for the installed app.')
+  .parse();
+const options = program.opts();
+
+type RepoId = string;
+type DateIsoString = string;
+interface IndexData {
```

Comment:
`IndexData` is a bit difficult to understand, in my opinion. Maybe something like `RepositoryInfoStatus`. I understood that it's the `index.json` content, but using a better name can help to understand what this file contains.